bundles/org.openhab.binding.enigma2/src/main/java/org/openhab/binding/enigma2/handler/Enigma2Handler.java

```diff
@@ -47,22 +48,35 @@
 @NonNullByDefault
 public class Enigma2Handler extends BaseThingHandler {
     private final Logger logger = LoggerFactory.getLogger(Enigma2Handler.class);
+    @Nullable
     private Enigma2Configuration config;
+    @Nullable
     private Enigma2Client enigma2Client;
     @Nullable
     private ScheduledFuture<?> refreshJob;
 
     public Enigma2Handler(Thing thing) {
         super(thing);
-        config = getConfigAs(Enigma2Configuration.class);
-        enigma2Client = new Enigma2Client(config.host, config.user, config.password, config.timeout);
     }
 
     @Override
+    @SuppressWarnings("null")
     public void initialize() {
+        config = getConfigAs(Enigma2Configuration.class);
+        if(config.host == null || config.host.isEmpty()) {
+            throw new IllegalArgumentException("host must not be null or empty");
```

Comment:
`initialize` should not throw any exceptions, please set the thing to OFFLINE with the apropriate ThingStatusDetail and maybe a message for the user.

Reply:
Fixed